docs/requirements.md

```diff
@@ -2,6 +2,12 @@
 
 This document explains the requirements and recommendations in the underlying databases of ScalarDB to make ScalarDB applications work correctly.
 
+## Common requirements
+
+### Privileges to access the underlying databases
+
+ScalarDB operates the underlying databases not only for CRUD operations but also for creating/altering schema, tables, indexes, and so on. Thus, ScalarDB requires a fully privileged account to access the underlying databases.
```

Comment:
I'd like to avoid `and so on` here since the phrase doesn't provide any value to the reader. How about we rephrase this part as suggested below?:

```suggestion
ScalarDB operates the underlying databases not only for CRUD operations but also for performing operations like creating or altering schemas, tables, or indexes. Thus, ScalarDB requires a fully privileged account to access the underlying databases.
```